File: helfomat-import/src/main/java/de/helfenkannjeder/helfomat/config/ImporterConfiguration.java
package de.helfenkannjeder.helfomat.config;

import org.springframework.boot.context.properties.ConfigurationProperties;

/**
 * @author Valentin Zickner
 */
@ConfigurationProperties("helfomat.importer")
public class ImporterConfiguration {

    private String webApiUrl;

    public String getWebApiUrl() {
        return webApiUrl;
    }

    public void setWebApiUrl(String webApiUrl) {
        this.webApiUrl = webApiUrl;
    }
}

File: helfomat-import/src/main/kotlin/de/helfenkannjeder/helfomat/config/ImporterConfiguration.kt
package de.helfenkannjeder.helfomat.config

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.ConstructorBinding

/**
 * @author Valentin Zickner
 */
@ConstructorBinding
@ConfigurationProperties("helfomat.importer")
class ImporterConfiguration (
    var webApiUrl: String
)